proseco/tests/test_guide.py

```diff
@@ -316,15 +316,31 @@ def test_check_spoiler_cases():
             stars = StarsTable.empty()
             stars.add_fake_star(row=r, col=c, mag=mag0, id=1, ASPQ1=1)
             # Add a "spoiling" star 5 mags fainter and move it from center out through a corner
-            stars.add_fake_star(row=r + drc, col=c + drc, mag=mag0 + dmag, id=2, ASPQ1=0)
+            stars.add_fake_star(row=r + drc, col=c + drc, mag=mag0 + dmag, id=2, ASPQ1=30)
             selected = get_guide_catalog(**STD_INFO, stars=stars, dark=dark)
             spoiled.append(1 if (1 not in selected['id']) else 0)
     spoiled = np.array(spoiled).reshape(-1, len(drcs)).tolist()
     #                    0  2  4  6  8 10 12 pixels
     expected_spoiled = [[1, 1, 1, 1, 0, 0, 0],  # dmag = 3
                         [1, 1, 1, 1, 0, 0, 0],  # dmag = 5
                         [1, 1, 1, 1, 0, 0, 0]]  # dmag = 7
+    assert spoiled == expected_spoiled
 
+    # Add a brighter "spoiler" that is a good guide star and confirm the test for
```

Comment:
Put this into a new test function, i.e. try to make tests conceptually narrow. If worried about repeating yourself in setup this can be handled with pytest fixtures, though in this case it is easier to just repeat the necessary setup.

Reply:
OK.  I thought the "spoiling" still was conceptually narrow, but no trouble to break out.